handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,24 @@
 
 ### Long term
 
-**_Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users._**
+**_The web team has three areas of focus: creating and maintaining a highly usable and intentionally designed webapp interface; delivering the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users; and developing code insights into an entirely new featureset._**
 
-**Outcome**: Our webapp, browser extensions and native integrations are great platforms to provide our users unique value through extensions. These platforms are well-maintained, consistent, easy to setup, well-documented, well-tested, performant, and show their power through convincing extensions built on top of them. The extensions platform and API provide powerful capabilities to extension developers and a great developer experience.
+**Outcome**: Our webapp, browser extensions and native integrations, and code insights all provide our users unique value. These platforms are well-maintained, well-designed, consistent, easy to setup, well-documented, well-tested, performant, and leave users impressed. The webapp helps users discover and employ the full power of Sourcegraph effectively, with a high quality and highly usable interface. The extensions and extensions API provide powerful capabilities to users and a great experience for extension developers. Code insights expose the value of code metadata to users at all levels of an organization. 
```

Comment:
```suggestion
**Outcome**: 
1. Our webapp, browser extensions and native integrations, and code insights all provide our users unique value. 
1. These platforms are well-maintained, well-designed, consistent, easy to setup, well-documented, well-tested, performant, and leave users impressed. 
1. The webapp helps users discover and employ the full power of Sourcegraph effectively, with a high quality and highly usable interface. 
1. The extensions and extensions API provide powerful capabilities to users and a great experience for extension developers. 
1. Code insights expose the value of code metadata to users at all levels of an organization. 
```

**suggestion (non-blocking)**: Propose we make this a list so that it's easier to digest.